File: VelibStations_Android/app/src/main/java/com/muhammedsafiulazam/velibstations/feature/stationlist/event/StationListEventType.kt
package com.muhammedsafiulazam.velibstations.feature.stationlist.event

object StationListEventType {
    // Tag.
    private const val TAG = "STATION_LIST_EVENT_TYPE_"

    // Event types.
    const val VIEWMODEL_REQUEST_LOAD_DATA: String = TAG + "VIEWMODEL_REQUEST_LOAD_DATA"
    const val VIEWMODEL_RESPONSE_LOAD_DATA: String = TAG + "VIEWMODEL_RESPONSE_LOAD_DATA"

    const val MODEL_REQUEST_LOAD_DATA: String = TAG + "MODEL_REQUEST_LOAD_DATA"
    const val MODEL_RESPONSE_LOAD_DATA: String = TAG + "MODEL_RESPONSE_LOAD_DATA"
}

File: VelibStations_iOS/VelibStations/Feature/StationList/Event/StationListEventType.swift
//
//  StationListEventType.swift
//  VelibStations
//
//  Created by Safi on 07/10/2019.
//

import Foundation

class StationListEventType {
    // Tag.
    private static let TAG: String = "STATION_LIST_EVENT_TYPE_"
    
    // Event types.
    static let VIEWMODEL_REQUEST_LOAD_DATA: String = TAG + "VIEWMODEL_REQUEST_LOAD_DATA"
    static let VIEWMODEL_RESPONSE_LOAD_DATA: String = TAG + "VIEWMODEL_RESPONSE_LOAD_DATA"
    
    static let MODEL_REQUEST_LOAD_DATA: String = TAG + "MODEL_REQUEST_LOAD_DATA"
    static let MODEL_RESPONSE_LOAD_DATA: String = TAG + "MODEL_RESPONSE_LOAD_DATA"
}
